packages/react-mutation-mapper/src/component/lollipopMutationPlot/LollipopMutationPlot.tsx

```diff
@@ -556,10 +556,76 @@ export default class LollipopMutationPlot<
 
     @autobind
     private getSVG(): SVGElement {
-        let svg: SVGElement = $(this.divContainer).find(
+        let lollipopSvg: SVGElement = $(this.divContainer).find(
             '.lollipop-svgnode'
         )[0] as any;
-        return svg;
+
+        const SVG_NS = 'http://www.w3.org/2000/svg';
+        const TEXT_STYLE = 'font-size: 12px; font-family:Arial;';
+        const TRACK_INDENT = 16;
+        const SUBTRACK_INDENT = TRACK_INDENT + 12;
+
+        function createTextTag(textNode?: Text, x: number = 0, y: number = 0) {
+            const text = document.createElementNS(
+                SVG_NS,
+                'text'
+            ) as SVGTextElement;
+            $(text).attr({ style: TEXT_STYLE, x, y });
+            if (textNode) {
+                text.append(textNode);
+            }
+            return text;
+        }
+
+        function createGroupTag(transform?: string) {
+            const group = document.createElementNS(SVG_NS, 'g') as SVGGElement;
+            group.setAttribute('transform', transform || '');
+            return group;
+        }
+
+        var outerSvg = document.createElementNS(SVG_NS, 'svg') as SVGElement;
+        outerSvg.append(lollipopSvg.cloneNode(true));
```

Comment:
put the clone operation outside of the append to so you can write a comment like ```clone the chart for use in download```